gui/src/2D/controls/scrollViewer.ts

```diff
@@ -361,4 +397,28 @@ export class ScrollViewer extends Rectangle {
             this._updateScroller(windowContents);
         });
     }
+
+    /** @hidden */
+    private _attachWheel() {
+        let scene = this._host.getScene();
+        scene!.onPointerObservable.add((pi, state) => {
+            if (!this._pointerIsOver || pi.type !== BABYLON.PointerEventTypes.POINTERWHEEL) {

```

Comment:
This is the observers I do not see disposed

Reply:
... my bad I was looking at the constructor ones, on it

Reply:
Done And Merged, it will be in the next nightly. Thanks a ton.